config/routes.php

```diff
@@ -93,6 +93,13 @@
         ['controller' => 'Model', 'action' => 'view'],
         ['pass' => ['id'], '_name' => 'model:view']
     );
+
+    $routes->connect(
+        '/model/:resource_type/savePropertiesJson',
```

Comment:
can `:resource_type` be other than `property_types` ? 
we could set explicitly ```['resource_types' => 'property_types']```

Reply:
yes it could used also for object_types and relation